pkg/config/containers.conf

```diff
@@ -409,6 +409,10 @@ default_sysctls = [
 #
 #events_logger = "journald"
 
+# Creates an additional audit event when creating a container.  The event
+# includes a JSON payload with detailed information about the container.
+#events_audit_container_create = true
```

Comment:
I think the commented line should always reflect the default value which is false.